core/templates/pages/oppia-root/routing/app.routing.module.ts

```diff
@@ -59,6 +59,14 @@ const routes: Route[] = [
       .then(m => m.EmailDashboardPageModule),
     canActivate: [IsLoggedInGuard]
   },
+  {
+    path: AppConstants.PAGES_REGISTERED_WITH_FRONTEND
+      .DIAGNOSTIC_TEST_PLAYER.ROUTE,
+    loadChildren: () => import(
+      'pages/diagnostic-test-player-page/diagnostic-test-player-page.module')
+      .then(m => m.DiagnosticTestPlayerPageModule),
+    canActivate: [IsLoggedInGuard]
```

Comment:
Do we need to be loggedIn?

Reply:
It looks like we don't need to be logged in to access the page. So I removed the auth guard.

Reply:
Done